conjure-java-jackson-serialization/src/main/java/com/palantir/conjure/java/serialization/ObjectMappers.java

```diff
@@ -113,10 +126,68 @@ public static ObjectMapper newCborServerObjectMapper() {
      *   <li>Throw on unknown properties found during deserialization.
      * </ul>
      */
+    public static SmileMapper newServerSmileMapper() {
+        return withDefaultModules(SmileMapper.builder(SMILE_FACTORY))
+                .enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+                .build();
+    }
+
+    public static ObjectMapper newClientObjectMapper() {
```

Comment:
Would like to deprecate these methods. I think we can fix this automatically with the `@InlineMe` annotation. Do you know if our infra automatically applies suggested fixes for that check?

http://errorprone.info/docs/inlineme

Reply:
It does not -- we'd need to handle running fixes on all dep updates in addition to baseline upgrades, otherwise some excavators would fail.

I'm not sure inline-me would produce code that actually compiles, it depends how the result is used. Prefer to handle automated migrations in isolation before deprecation